### PR TITLE
Board Specific changes 

### PR DESCRIPTION
**Pull request by johnnavaj**
Currently i have added the required changes, testing is still going on from our side . Feel free ask if any issue or concern is there.

### CODE REVIEW

File: boards/artix7_100t.json [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
+        "server": {
+          "arguments": [
+            "-f", "~/.platformio/packages/framework-shakti-sdk/bsp/third_party/artix7_100t/ftdi.cfg"
+          ],
+          "executable": "~/.platformio/packages/tool-openocd/openocd",
+	  "package": "tool-openocd"
+        }
+
+      }
+    }
```
Comment (valeros) [inline boards/artix7_100t.json]: Already implemented [here](https://github.com/platformio/platform-shakti/blob/develop/platform.py#L52-L53)

File: boards/artix7_100t.json [hunk 2]
```diff
@@ -1,31 +1,41 @@
...
   "frameworks": [
     "shakti-sdk"
   ],
   "name": "Arty A7-100: Artix-7 FPGA Development Board",
   "upload": {
-    "maximum_ram_size": 268435456,
-    "maximum_size": 16777216,
+"maximum_size":0,
+      "maximum_ram_size": 134217728,
+    "speed" : 19200,
```
Comment (valeros) [inline boards/artix7_100t.json]: What exactly does speed field represent here? Will it be used in the upload routine? Is there only RAM memory on board? Any flash memory?
Also, the formatting is broken.

File: boards/artix7_100t.json (hunk 1 above)
Comment (valeros) [reply boards/artix7_100t.json]: You can add the path to your SVD file in the debug section, for example like this [one](https://github.com/platformio/platform-ststm32/blob/develop/boards/b96b_f446ve.json#L20)

Comment (johnnavaj) [reply boards/artix7_100t.json]: Done.

File: boards/artix7_100t.json (hunk 2 above)
Comment (johnnavaj) [reply boards/artix7_100t.json]: fixed formatting , we are currently testing the upload command as there is some confusion from the hardware side.

Comment (johnnavaj) [reply boards/artix7_100t.json]: For 35t we don't have RAM(DDR integrated) its written to tcm as part of the fpga configuration. For 100t we have DDR as a slandered.

File: boards/artix7_35t.json [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
-    "mcu": "e31"
+    "mcu": "E-Class"
   },
   "debug": {
-    "onboard_tools": [
-      "ftdi"
-    ]
+
+	 "onboard_tools": ["ftdi"],
+         "svd_path":"arty35t.svd"
```
Comment (valeros) [inline boards/artix7_35t.json:19]: Please use formatting that is consistent with other parts of the file: no empty lines, 2 whitespace characters for indentation.

File: boards/artix7_35t.json [hunk 4]
```diff
@@ -1,31 +1,32 @@
...
   "frameworks": [
     "shakti-sdk"
   ],
   "name": "Artix-7 35T Arty FPGA Evaluation Kit",
   "upload": {
-    "maximum_ram_size": 268435456,
-    "maximum_size": 16777216,
+    "maximum_ram_size": 0,
+    "maximum_size": 131072,
+    "speed" : 19200,
```
Comment (valeros) [inline boards/artix7_35t.json]: I'm a bit confused about these values.
Where is `speed` value used? I see you specified monitor speed in examples, so why do you specify this value in upload section again?
`maximum_ram_size: 0` How do you represent volatile memory in this case? I see there is `256 MB DDR3L` on board, is it used in any way?
`maximum_size: 131072` Any reason why it's exactly 128K? There is `16 MB Quad-SPI Flash` in the specification for this board.

Comment (johnnavaj) [reply boards/artix7_35t.json]: Speed i have excluded as i got confused between monitor speed and the speed. For the RAM confusion, we have configured only 128MB (arty_100t) usable as part of the bitstream. Also for 35t there is a TCM of 128KB , but for this case can i put it as part of the "minimum_ram_size"?

Comment (johnnavaj) [reply boards/artix7_35t.json]: Details for memory configuration of both boards 

Arty 100t
RAM- 128MB DDR(128MB is usable out of 256MB )
SPI-Not integrated

Arty 35t
RAM- TCM configured upto 128KB
SPI-Not integrated

Comment (johnnavaj) [reply boards/artix7_35t.json]: @valeros

Comment (valeros) [reply boards/artix7_35t.json]: Thanks for the clarification, let's specify 128MB as `maximum_ram_size` for `Arty 100t` and 128K as  `maximum_ram_size` for `Arty 35t`.
Also, do you mind creating a clean pull request as this one has a lot of "fix" commits. Please and make it consistent with our formatting.

Comment (johnnavaj) [reply boards/artix7_35t.json]: Agreed , once i fix the formatting i will close this and create a new PR. One more thing , what about the maximum_size variable ? as we don't have an SPI integrated now. Put it there or other ?

Comment (valeros) [reply boards/artix7_35t.json]: Thanks, `maximum_size: 0` should be fine

Comment (johnnavaj) [reply boards/artix7_35t.json]: Cool, Will do . Thanks!

**Pull request closed without merging**